Library/Formula/slacknimate.rb

```diff
@@ -0,0 +1,24 @@
+class Slacknimate < Formula
+  desc "text animation for Slack messages"
+  homepage "https://github.com/mroth/slacknimate"
+  url "https://github.com/mroth/slacknimate/archive/v1.0.0.tar.gz"
+  sha256 "7d495e2233a379f3ffffb2445f4b41b4306e52049d71f4c4c873786bb4048f45"
+  head "https://github.com/mroth/slacknimate.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/mroth"
+    ln_s buildpath, buildpath/"src/github.com/mroth/slacknimate"
+    system "glide", "install"
```

Comment:
> As a sidenote, is that TODO around build-dep caching documented somewhere public? I'd be potentially interested in getting involved and helping out.

I don’t think we have anything documented on that, we talked a bit about it but that’s all. Maybe @mikemcquaid or @DomT4 have ideas on that?